tests/system/testdata/sessions/stop.txtar

```diff
@@ -26,6 +26,22 @@ ak sessions log ses_00000000000000000000000004 --skip -1 --no-timestamps -j
 return code == 0
 output equals_json file last.json
 
+ak deploy --manifest test.yaml --file test.star
```

Comment:
How can I run just this test?

Reply:
@tebeka https://github.com/autokitteh/autokitteh/blob/main/tests/system/README.md#execution-cheatsheet